ports/libmodplug/portfile.cmake

```
@@ -23,15 +24,6 @@ vcpkg_install_cmake()
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL dynamic AND VCPKG_TARGET_IS_WINDOWS)
-    if(VCPKG_TARGET_IS_MINGW)
-        set(BIN_NAME libmodplug.dll)
-    else()
-        set(BIN_NAME modplug.dll)
-    endif()
-    file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/bin)
-    file(RENAME ${CURRENT_PACKAGES_DIR}/lib/${BIN_NAME} ${CURRENT_PACKAGES_DIR}/bin/${BIN_NAME})
-    file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/debug/bin)
-    file(RENAME ${CURRENT_PACKAGES_DIR}/debug/lib/${BIN_NAME} ${CURRENT_PACKAGES_DIR}/debug/bin/${BIN_NAME})
     vcpkg_copy_pdbs()
 endif()

```

```suggestion
vcpkg_copy_pdbs()
```
We don't need the judgment now.